src/debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -305,6 +305,28 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
             return debugConfiguration.configureAspNetCoreSsl;
         }
 
+        const launchSettingsPath = path.join(path.dirname(projectFile), 'Properties', 'launchSettings.json');
+
+        if (await this.fsProvider.fileExists(launchSettingsPath)) {
+            let launchSettingsString = await this.fsProvider.readFile(launchSettingsPath);
+            if (launchSettingsString.charCodeAt(0) === 0xFEFF) {
```

Comment:
Not really sure why this is necessary.

Also rather than read _then_ parse, we typically use `fse.readJSON` from 'fs-extra' which does them in the same step. It's simpler and I _think_ more robust, but not entirely sure

Reply:
I'll try it. I had to read/parse separately with JSON.parse because it couldn't comprehend what a byte order mark was.

Reply:
Looks like fse.readJson is smart enough to deal with BOM so I'll definitely use that.